language/move-core/types/src/vm_status.rs

```diff
@@ -660,6 +660,8 @@ pub enum StatusCode {
     CALL_STACK_OVERFLOW = 4021,
     VM_MAX_TYPE_DEPTH_REACHED = 4024,
     VM_MAX_VALUE_DEPTH_REACHED = 4025,
+    VM_EXTENSION_ERROR = 4026,
```

Comment:
Could you clarify exactly what this error stands for? Just want to understand whether it's supposed to be a user error or an internal error/invariant violation.